core/src/integrationTest/java/tech/pegasys/eth2signer/core/DirectoryBackedArtifactSigningProviderIntegrationTest.java

```diff
@@ -64,14 +61,16 @@
   @BeforeEach
   void setup() {
     vertx = Vertx.vertx();
-    hashicorpConnectionFactory = new HashicorpConnectionFactory(vertx);
+    final HashicorpConnectionFactory hashicorpConnectionFactory =
+        new HashicorpConnectionFactory(vertx);
 
-    artifactSignerFactory =
+    final ArtifactSignerFactory artifactSignerFactory =
         new ArtifactSignerFactory(
             configsDirectory, new NoOpMetricsSystem(), hashicorpConnectionFactory);
-    signerParser = new YamlSignerParser(artifactSignerFactory);
+    final SignerParser signerParser = new YamlSignerParser(artifactSignerFactory);
     signerProvider =
-        new DirectoryBackedArtifactSignerProvider(configsDirectory, FILE_EXTENSION, signerParser);
+        new DirectoryBackedArtifactSignerProvider(
+            configsDirectory, FILE_EXTENSION, signerParser, 0);
```

Comment:
This 0 is cache size?

Reply:
Yeah 0 for cache size disables the cache, I don't want the cache interfering with the integration tests.